scripts/ci_ecs_benchmarks.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# assign local ip
+ip=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
+webserver_url=http://"$ip":9000
+da_webserver_url=http://"$ip":9001
+orchestrator_url=http://"$ip":4444
+
+# start webserver
+just async_std example webserver -- http://0.0.0.0:9000 &
+just async_std example webserver -- http://0.0.0.0:9001 &
+sleep 10m
+
+# start orchestrator
+just async_std example orchestrator-webserver -- --config_file ./crates/orchestrator/run-config.toml --orchestrator_url http://0.0.0.0:4444 --webserver_url ${webserver_url} --da_webserver_url ${da_webserver_url} &
+sleep 1m
+
+# start validators
+ecs deploy --region us-east-2 hotshot hotshot_centralized -i centralized ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+ecs deploy --region us-east-2 hotshot hotshot_centralized -c centralized ${orchestrator_url}
+ecs scale --region us-east-2 hotshot hotshot_centralized 10 --timeout -1
+sleep 2m #rethink about this
```

Comment:
Looks like in `scripts/aws_ecs_benchmarks.sh`, this is set to be proportional to the number of rounds and nodes. Curious why we don't do the same here.

Reply:
Yeah we should do the same here, but just ignore this file, it's mostly for reference and not in use now :( I'll have to re-think each step of this file when we do real CI integration.